packages/ui/src/lib/button/Button.stories.svelte

```diff
@@ -1,16 +1,51 @@
 <script>
-    import {Meta, Template, Story} from '@storybook/addon-svelte-csf';
+	import { ArrowCircleDownIcon } from '@rgossiaux/svelte-heroicons/solid';
 
-    import Button from "./Button.svelte";
+	import { Meta, Story, Template } from '@storybook/addon-svelte-csf';
 
+	import Button from './Button.svelte';
 </script>
 
-<Meta title="Ui/Button" component={Button}/>
+<Meta title="Ui/Button" component={Button} />
 
 <Template let:args>
-    <Button {...args}>
-        I'm a button from UI!
-    </Button>
+	<Button {...args}>I'm a button from UI!</Button>
 </Template>
 
 <Story name="Default" />
+
+<Story name="Primary">
+	<Button style="primary">Primary Style</Button>
+</Story>
```

Comment:
```suggestion
<Story name="Primary">
    <Button style="primary">Primary Style</Button>
    <Button style="primary" disabled>Primary Style (disabled)</Button>
</Story>
```

Show the enabled/disabled states for each variant side-by-side.